.gitignore

```diff
@@ -47,3 +47,6 @@ profile.cov
 /dashboard/assets/package-lock.json
 
 **/yarn-error.log
+
+# datadir
+*datadir*
```

Comment:
What's this folder?

Reply:
will have an `l2gethdatadir` locally, if running a node.

Reply:
I always specify the datadir using `--datadir abc`

Reply:
got it. removed this line in `.gitignore`.